PhaImportNotifications/Endpoints/ImportNotificationEndpoint.cs

```diff
@@ -0,0 +1,57 @@
+using Microsoft.AspNetCore.Mvc;
+using Trade.ImportNotification.Contract;
+
+namespace PhaImportNotifications.Endpoints;
+
+public static class ImportNotificationUpdatesEndpoint
+{
+    public static void UseImportNotificationEndpoints(this IEndpointRouteBuilder app)
+    {
+        app.MapGet("import-notifications/{referenceNumber}/", Get);
+        app.MapGet("import-notifications-updates/{port}/", GetAll);
```

Comment:
Don't think port will be in here like this. Ports will be translated from the client that is calling us won't they?

Reply:
Agreed to rename.